cumulus/parachains/runtimes/testing/penpal/src/lib.rs

```diff
@@ -494,7 +495,10 @@ impl pallet_assets::Config<ForeignAssetsInstance> for Runtime {
 	type AssetId = ForeignAssetsAssetId;
 	type AssetIdParameter = ForeignAssetsAssetId;
 	type Currency = Balances;
-	type CreateOrigin = AsEnsureOriginWithArg<EnsureSigned<AccountId>>;
+	// This is to allow any other remote location to create foreign assets. Used in tests, not
+	// recommended on real chains.
+	type CreateOrigin =
+		ForeignCreators<Everything, LocationToAccountId, AccountId, xcm::latest::Location>;
```

Comment:
This change breaks existing tests. I'm planning on implementing this trait for tuples and putting a tuple here with the old value.

Reply:
Which test does it break and why?

Reply:
Tests that create foreign assets on penpal with an account origin

Reply:
In `bridge-hub-rococo-integration-tests` for example, `send_token_from_ethereum_to_penpal`

Reply:
looking into all emulated tests now

Reply:
this one broke it: https://github.com/paritytech/polkadot-sdk/pull/4826/commits/abb8142b2cec0538cbe50481b142f27a6b6b67a8

got messed up when you fixed conflicts, merge damage is one of the most pervasive errors - because people don't usually review merge commits and changes can be overlooked

I suggest you run integ tests before and after fixing conflicts on merges to catch errors